src/main/java/frc/robot/RobotContainer.java

```diff
@@ -404,6 +403,7 @@ public void setAutoPlacementLEDs(Optional<Alliance> alliance) {
 
       subLEDs.setIndividualLED(YTranslationColor, 5);
       subLEDs.setIndividualLED(YTranslationColor, 6);
+      subLEDs.clearAnimationChunk(8, 0);
```

Comment:
Before testing, set the count parameter to be greater than 0 (maybe 50-100 so you could actually see it do something). Otherwise, you're saying "apply this color to 0 LEDs!" and it won't do anything :(
While testing, try to find the actual number of LEDs that it will take to clear a whole strip.

Reply:
On another note, do we want to be sending requests to the LEDs every loop? It would save processing power if we only sent requests to the LEDs (especially clearing the animation) if there is a change in their desired color. In this case, we would only clear the animation if the current status of the LEDs is not clear

Reply:
hehe "note." But yes that would be better. I just don't have a clear (hehe) plan on how to do that at the moment.

Reply:
References in new issue